source/docs/05-mensural.xml

```diff
@@ -269,7 +269,7 @@
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/mensuration_changes.mei" parse="text"/></egXML>
                      </figure>
                   </p>
-                  <p><hi rend="bold">Sesquialtera</hi> is frequently used to change the mensuration. The effect of the sesquialtera on the mensuration can be encoded by using the <att>tempus</att> and <att>prolatio</att> attributes of <gi scheme="MEI">mensur</gi> (for example, when changing the tempus to perfect, the effect can be encoded in <att>tempus="3"</att>). The actual sesquialtera can be encoded using <att>num="3"</att>, <att>numbase="2"</att>, and <att>level</att> to define the note level the sesquialtera is applied to (<abbr>e.g.</abbr>, <att>level="semibrevis"</att>).</p>
+                  <p><hi rend="bold">Sesquialtera</hi> is frequently used to change the mensuration. The effect of the sesquialtera on the mensuration can be encoded by using the <att>tempus</att> and <att>prolatio</att> attributes of <gi scheme="MEI">mensur</gi> (for example, when changing the tempus to perfect, the effect can be encoded in <att>tempus</att>="3"). The actual sesquialtera can be encoded using <att>num</att>="3", <att>numbase</att>="2", and <att>level</att> to define the note level the sesquialtera is applied to (<abbr>e.g.</abbr>, <att>level</att>="semibrevis").</p>
```

Comment:
Parallel cases use `<hi rend="italic">` for the complete attribute+value. Although I don't think that this is the best solution, we should keep consistency here.

Reply:
I see. Can you point me to some instances of this usage? I found the approach used here applied in the CMN chapter and double checked with the rendering in the PDF.

`<att>tempus</att>="3"` renders as `@tempus="3"`
while `<hi rend="italic">tempus="3"` renders as _tempus="3"_

Reply:
[Here](https://github.com/music-encoding/music-encoding/blob/3efe83de4dba12c96caec61b23c973f361b396ec/source/docs/05-mensural.xml#L192C1345-L192C1345) and [here](https://github.com/music-encoding/music-encoding/blob/3efe83de4dba12c96caec61b23c973f361b396ec/source/docs/05-mensural.xml#L202C1-L202C1). But, as you pointed out, in chapters 4 and 9 it's your proposed way, so maybe the others should be changed …

Reply:
I forgot line 272: https://github.com/music-encoding/music-encoding/blob/3efe83de4dba12c96caec61b23c973f361b396ec/source/docs/05-mensural.xml#L272

Reply:
Thanks for the pointers. Adjusted it in chapter 5.